### PR TITLE
updates templates samples and fixed comments 

### PR DESCRIPTION
**Pull request by mksuni**

### CODE REVIEW

File: PYTHON/AzureSQLDB/read_from_azure_sql_db.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This sample allows you to read data from Azure SQL database 
+# Complete these steps before testing this funtion 
+#   1. Select Manage connections to connect to Azure SQL database 
+#   2. Copy the Alias name and replace it below
+
+@app.connection("sqlDB",alias="<alias for azure sql database>")
```
Comment (marshall-lerner) [inline PYTHON/AzureSQLDB/read_from_azure_sql_db.py:6]: This attribute (in this sample and in all others) is currently called `app.fabric_item_input(...)`, but might soon be changing to `connection`

File: PYTHON/index.json [hunk 2]
```diff
@@ -1,42 +1,94 @@
...
-          }
-        ]
-    }
+[
+   {
+      "name":"Warehouse",
+      "description":"Sample functions to read from and write data to a warehouse in Fabric.",
+      "data":[
+         {
+            "name":"export data from warehouse into a lakehouse",
```
Comment (calvinhsia) [inline PYTHON/index.json:7]: capital? 'Export'

File: PYTHON/index.json [hunk 3]
```diff
@@ -1,42 +1,94 @@
...
+            "description":"Use this user data function that write data from a warehouse into csv file and saves it in a lakehouse",
```
Comment (calvinhsia) [inline PYTHON/index.json:8]: 'that writes'

File: PYTHON/index.json [hunk 4]
```diff
@@ -1,42 +1,94 @@
...
+            "detail":"Add a warehouse and lakehouse connection. Then use this sample function that write data from a warehouse into csv file and saves it in a lakehouse",
```
Comment (calvinhsia) [inline PYTHON/index.json:9]: into a csv

File: PYTHON/index.json [hunk 5]
```diff
@@ -1,42 +1,94 @@
...
+   },
+   {
+      "name":"Lakehouse",
+      "description":"Sample functions to work with tables and files within a lakehouse in Fabric.",
+      "data":[
+         {
+            "name":"Write csv file into a lakehouse",
+            "description":"TThis sample writes a CSV file into a lakehouse  using pandas.",
+            "dateAdded":"2024-11-05T00:00:00Z",
+            "data":"Lakehouse/manipulate_data_with_pandas.py"
```
Comment (calvinhsia) [inline PYTHON/index.json:30]: does this file exist?

File: PYTHON/AzureSQLDB/read_from_azure_sql_db.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+@app.function("read_from_azure_sql_db")
+def read_from_azure_sql_db(sqlDB: udf.FabricSqlConnection)->str:
```
Comment (marshall-lerner) [inline PYTHON/AzureSQLDB/read_from_azure_sql_db.py:8]: similarly with udf.*, the templated currently have `fabric.functions.*`. We sound like we'll move to udf.* soon, so we could either wait until the train that updates the template gets to prod and then merge this PR, or we could merge this PR beforehand if we include the import `import fabric.functions as udf` at the top of each of these samples

File: PYTHON/Lakehouse/query_data_from_tables.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
+# This sample reads data from a table in a lakehouse 
+# Complete these steps before testing this function
+# 1. Select Manage connections and add a connection to a Lakehouse 
+
+import datetime
+import json
+
+@app.fabric_item_input(argName="mylakehouse", alias="<My Lakehouse alias>")
```
Comment (marshall-lerner) [inline PYTHON/Lakehouse/query_data_from_tables.py:8]: some samples currently use app.connection, and others use fabric_item_input, so we should make them consistent when we decide which to use

File: PYTHON/Lakehouse/query_data_from_tables.py [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+@app.function("query_data_from_tables")
+def query_data_from_tables(mylakehouse: fabric.functions.FabricSqlConnection) -> str:
```
Comment (marshall-lerner) [inline PYTHON/Lakehouse/query_data_from_tables.py:10]: same with fabric.functions.*

File: PYTHON/AzureSQLDB/read_from_azure_sql_db.py (hunk 1 above)
Comment (mksuni) [reply PYTHON/AzureSQLDB/read_from_azure_sql_db.py]: @marshall-lerner  yes , for now I shall keep this and will change once your changes are in

**Pull request merged**